### PR TITLE
[10]Create & Style the Customer Signup Page using CSS

### PR DESCRIPTION
**Pull request by kazeemerinfolami**
## Task: `[10]Create & Style the Customer Signup Page using CSS`
- `Customer Signup Page using CSS `
Screen Shot 2020-10-29 at 5 28 23 PM
Screen Shot 2020-10-28 at 9 09 26 AM
Screen Shot 2020-10-29 at 5 28 46 PM
Screen Shot 2020-10-28 at 9 09 50 AM

### CODE REVIEW

**Comment by adaugochi** [comment]
Well done @kazeemerinfolami 

Kindly fix codefactor issue on your PR for it to be ready for review

**Review by kazeemerinfolami — COMMENTED** [review]
code factor issue fixed

**Comment by kazeemerinfolami** [comment]
I have made the correction please check it.

File: submissions/kazeemErinfolami/htmlForms/signUp.html [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="../css/signup.css">
+    <title>Sign up</title>
+  </head>
+  <body>
+    <nav>
+      <label class="logo"><img id="logo" src="../html/images/logo.png"></label>
+      <ul>
+        <li><a href="../html/index.html">Product Page</a></li>
+        <li><a href="#">Sign In</a></li>
```
Comment (adaugochi) [inline submissions/kazeemErinfolami/htmlForms/signUp.html:14]: Do you mean Sign Up?

**Review by adaugochi — CHANGES_REQUESTED** [review]
Well done @kazeemerinfolami 

For things like `margin`, `left`, `padding`, `top` `right`, `bottom`, kindly use `px` or `rem` using `%` is a red flag

File: submissions/kazeemErinfolami/htmlForms/signUp.html (hunk 1 above)
Comment (kazeemerinfolami) [reply submissions/kazeemErinfolami/htmlForms/signUp.html:14]: its a link to the sign-in page

**Review by kazeemerinfolami — COMMENTED** [review]
fixed

**Comment by kazeemerinfolami** [comment]
Good morning, please check my PR

File: submissions/kazeemErinfolami/htmlForms/signUp.html (hunk 1 above)
Comment (adaugochi) [reply submissions/kazeemErinfolami/htmlForms/signUp.html:14]: The mockup says `Sign Up`. Here is an image of it 
image

**Review by adaugochi — CHANGES_REQUESTED** [review]
Kindly use `px` or `rem` as a unit for `font-size`

**Review by kazeemerinfolami — COMMENTED** [review]
fixed

**Review by adaugochi — APPROVED** [review]

**Pull request merged**